README.md

```diff
@@ -53,35 +90,27 @@ pod 'RNPurchases', :path => '../node_modules/react-native-purchases'
 
 In your `ios` folder, run `pod install`. If you've just upgraded ExpoKit, you might need to upgrade cocoapods to the newest version: `sudo gem install cocoapods`. 
 
-#### Migrating from manual installation (if your project doesn't use CocoapodsCreate)
+### Migrating from manual installation (if your project doesn't use CocoapodsCreate)
 
-##### Remove the Framework Reference from your project
+### Remove the Framework Reference from your project
 
 1. Remove `Purchases.framework` and `PurchasesHybridCommon.framework` from the libraries section of the project. 
 
-##### Remove iOS Frameworks to Embedded Binaries
+### Remove iOS Frameworks to Embedded Binaries
 1. In Xcode, in project manager, select your app target.
 1. Select the general tab
 1. Look for `Purchases.framework` and `PurchasesHybridCommon.framework` in the Embedded Binaries section and remove them.
 
 Remove `$(PROJECT_DIR)/../node_modules/react-native-purchases/ios` from Framework Search paths in build settings
 
-##### Remove Strip Frameworks Phase
+### Remove Strip Frameworks Phase
 During the old manual installation instructions, now deprecated, we indicated to add a build phase to strip fat frameworks. 
 1. In Xcode, in project manager, select your app target.
 2. Open the `Build Phases` tab
 3. Remove the added `Strip Frameworks` phase
 4. Clean `Derived Data` 
 
-##### Link static library
+### Link static library
```

Comment:
Same with all these other headers. We want them to be nested, removing these `#` will make them at the same level as  `Additional iOS Setup`

Reply:
Reverted.